go.mod

```diff
@@ -32,34 +25,29 @@ require (
 	github.com/ONSdigital/dp-net/v2 v2.11.2
 )
 
+require golang.org/x/sync v0.7.0 // indirect
+
 require (
+	github.com/ONSdigital/dp-s3/v2 v2.1.0
```

Comment:
Why is this in the indirect dependencies block?

Reply:
This is added automatically when running go get & go mod tidy.